### DISCUSSION TITLE
Proposal: Disabled pseudo class should be enabled by default

### DISCUSSION THREAD

**Discussion by alfonsobries** [tailwindlabs/tailwindcss#2234] [Ideas]
I have been working since the last year in a library of components with configurable tailwind classes (https://vue-tailwind.com/)

The idea is that the user defines the different variants of the elements an assign some classes to it, well... with all the experience I got working in this project I notice that almost in 100% of the cases the user needs to define a class for when an HTML form input is disabled,

That, of course, it's possible with a tailwind but requires to enable the `disabled` pseudo class or adding some JS logic for when the input is disabled

```
<button class="border p-3..." :class="{'opacity-50': isDisabled}" :disabled="isDisabled" >press me</button>
```

In summary, I think that pseudo-class is important enough to be enabled by default, thoughts?
